ibm/resource_ibm_is_instance_template.go

```diff
@@ -99,6 +99,14 @@ func resourceIBMISInstanceTemplate() *schema.Resource {
 				Description: "Profile info",
 			},
 
+			isInstanceTotalVolumeBandwidth: {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
```

Comment:
do we support update on this

Reply:
we do not support updated on this, so i have added ForceNew True.

Reply:
i have removed computed and kept only forcenew and optional